src/common/state/Variant.C

```diff
@@ -3822,3 +3843,113 @@ Variant::ConvertToString()
     return tmp;
 }
 
+// ****************************************************************************
+// Method: Variant::ConvertToPythonTupleString
+//
+// Purpose:
+//   Converts this object's data to a python tuple string representation.
+//
+// Programmer: Kathleen Biagas
+// Creation:   March 25, 2025 
+//
+// Modifications:
+//
+// ****************************************************************************
+
+string &
+Variant::ConvertToPythonTupleString()
+{
+    tmp.clear();
+    char retval[5000];
+    tmp += "(";
+    if (dataType == BOOL_VECTOR_TYPE)
```

Comment:
should there be an error case in this function?

Reply:
I'm not sure. That would mean every entry would have parens around it. I think it is cleaner the way it is.

Reply:
I added logic at top to return empty string if 'IsVector()' returns false. Will that do?

Reply:
Yup.

Reply:
I guess I wasn't paying close attention, sorry about that. An early return if this function is mistakenly called for something that is not a vector is good, but it might be better to have an error explaining that the function was called improperly as opposed to silently continuing. Also, if in the future we added more `*_VECTOR_TYPE`s, we'll have the same issue of silent failure. I was suggesting adding an `else` case to the conditional with an error message so that if this function is ever used improperly then we are safe. Would we ever add more `*_VECTOR_TYPE`s? I don't know; you would probably know better than me.

Reply:
There isn't any exception type handling elsewhere in the file nor is there any messaging for error type conditions. I don't know how/if such types of error handling/messaging would propagate up the pipeline. I think the empty string is sufficient, unless, since it is supposed to be a 'python-tuple-string', a string with simply '()' is returned as would be the result if the early return is removed.

Reply:
Sounds good.